_includes/v19.2/sql/physical-plan-url.md

```diff
@@ -0,0 +1 @@
+The physical query plan is encoded into a byte string after the [fragment identifier (`#`)](https://en.wikipedia.org/wiki/Fragment_identifier) in the generated URL. The fragment is not sent to the web server; instead, the browser waits for the web server to return a `decode.html` resource, and then JavaScript on the web page decodes the fragment into a physical query plan diagram. The query plan is, therefore, not logged by a server external to the CockroachDB cluster and not exposed unencoded to the public internet.
```

Comment:
You can remove that last "unencoded" - the query plan isn't exposed at all to the public internet.